### ISSUE TITLE
Pressing a modifier key sometimes interrupts slider movement

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/sun#431]
From https://github.com/phetsims/QA/issues/247, @rea-laura said

> When you use the keyboard nav to move the sliders, there are different key options to move in smaller and larger intervals. If you want to move in smaller intervals, you can use the arrow keys to move at the original speed, and then press and release shift to slow down the slider. You can press the keys in any order and the sim will respond accordingly. However, if you want to do the same thing with the larger intervals, you must press the function key and the arrow key at the same time for the larger interval to happen. If you press the arrow key first and then the function key, the sim will not respond and will keep moving the slider at the base speed. Is this a problem?

And then
> Additionally if you press the fn key first and then the arrow key, the slider will start moving at the larger interval. If you then release the fn key while keeping the arrow key down, the slider will stop moving entirely. This doesn't happen for the smaller intervals - you can press and release the keys in any order and the slider will respond accordingly.

It was reported this was observed on FF and Safari.

Making an issue because I think I am seeing something different on Chrome and I want to understand. It is likely this is just the native browser functionality for slider input and modifier keys vs dedicated keys (shift vs page up/page down).

**Comment by jessegreenberg** [comment]
On Chrome:
  - If I press the shift key while holding an arrow key, the slider stops moving until I press a key again.
  - If I press ANY key while pressing an arrow key, the slider stops moving until I press a key again.
  - If I press page up/page down while pressing an arrow key, the slider stops moving in larger steps. This is different, probably because I have a desktop keyboard with separate keys for page up/page down that don't use fn key.

**Comment by jessegreenberg** [comment]
I see the identical behavior to https://github.com/phetsims/sun/issues/431#issuecomment-452396505 in FF on Windows 10.

**Comment by jessegreenberg** [comment]
I see the identical behavior to https://github.com/phetsims/sun/issues/431#issuecomment-452396505 in Chrome in https://jsfiddle.net/jhedn16r/, so I am feeling more confident that this is normal behavior for the browser, and differences we are seeing are because of OS differences.

**Comment by jessegreenberg** [comment]
@rea-laura, are you able to see the same behavior you were seeing in the sim while using this vanilla HTML slider?

https://jsfiddle.net/jhedn16r/

**Comment by jessegreenberg** [comment]
Here is a new test that also changes step size on shift, only using modifier keys, similar to how it is being done in a PhET sim. @rea-laura do you observe the same things you were seeing in the original issue with this slider?
https://jsfiddle.net/jhedn16r/11/

**Comment by jessegreenberg** [comment]
Discussed with @rea-laura over slack. The original concern was that the shift key and page up/page down keys did not behave the same way. I think this is OK because these are classified as different key types so it makes sense that they will behave differently. The shift key is a "modifier" key, and you can press that in any order and expect it to work correctly. On the other hand, page up/page down are just regular keybaord keys (even though you access them through the fn key on some keyboards).

We verified that pressing any key while holding down an arrow key will interrupt the slider on both Windows and Mac.

@rea-laura wrote up a summary in the a11y quirk tracker so we can easily reference if this comes up again, so thanks! I think this issue can be closed.
